configs/platforms/osx-10.12-x86_64.rb

```diff
@@ -0,0 +1,14 @@
+platform "osx-10.12-x86_64" do |plat|
+  plat.servicetype 'launchd'
+  plat.servicedir '/Library/LaunchDaemons'
+  plat.codename "sierra"
+
+  plat.provision_with 'curl http://pl-build-tools.delivery.puppetlabs.net/osx/homebrew_sierra.tar.gz | tar -x --strip 1 -C /usr/local -f -'
```

Comment:
I'd like to refactor the way we use Homebrew a bit soon if possible, so should we roll running Brew based builds as someone other than root?